src/playback/qml/MuseScore/Playback/internal/MixerSoundSection.qml

```diff
@@ -47,7 +47,7 @@ MixerPanelSection {
             height: 26
 
             supportsByPassing: false
-            resourceItemModel: channelItem.inputResourceItem
+            resourceItemModel: channelItem.inputResourceItem ?? null
```

Comment:
ohh

Reply:
@Eism The bug is fixed in Qt Creator 8 :)